dev/scripts/install-vllm-cpu.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+# Create conda env
+# conda create -n vllm-cpu python=3.10
+# conda activate vllm-cpu
```

Comment:
Could these unnecessary comments be removed?

Reply:
I would change it to some notes for installing vllm-cpu for convenience.